src/test/java/uk/gov/hmcts/sptribs/common/notification/HearingPostponedNotificationTest.java

```diff
@@ -0,0 +1,189 @@
+        LocalDate expDate = LocalDate.now();
+        final CaseData data = getMockCaseData(expDate);
+        data.getCicCase().setRepresentativeFullName("repFullName");
+        data.getCicCase().setRepresentativeContactDetailsPreference(ContactPreferenceType.EMAIL);
+        data.getCicCase().setRepresentativeEmailAddress("testrepr@outlook.com");
+        data.getCicCase().setHearingList(getDynamicList());
+
+        //When
+        when(notificationHelper.buildEmailNotificationRequest(any(), anyMap(), any(TemplateName.class)))
+            .thenReturn(NotificationRequest.builder().build());
+        when(notificationHelper.getRepresentativeCommonVars(any(), any(CicCase.class))).thenReturn(new HashMap<>());
+        doNothing().when(notificationHelper).addHearingPostponedTemplateVars(any(CicCase.class), anyMap());
+        hearingPostponedNotification.sendToRepresentative(data, "CN1");
+
+        //Then
+        verify(notificationService).setNotificationRequest(any(NotificationRequest.class));
+
+        verify(notificationService).sendEmail();
+    }
+
+    @Test
+    void shouldNotifyRepresentativeWithPost() {
+        //Given
+        LocalDate expDate = LocalDate.now();
+        final CaseData data = getMockCaseData(expDate);
+        data.getCicCase().setRepresentativeFullName("repFullName");
+        data.getCicCase().setRepresentativeContactDetailsPreference(ContactPreferenceType.POST);
+        data.getCicCase().setRepresentativeAddress(AddressGlobalUK.builder().build());
+        data.getCicCase().setHearingList(getDynamicList());
+
+        //When
+        when(notificationHelper.buildLetterNotificationRequest(anyMap(), any(TemplateName.class)))
+            .thenReturn(NotificationRequest.builder().build());
+        when(notificationHelper.getRepresentativeCommonVars(any(), any(CicCase.class))).thenReturn(new HashMap<>());
+        doNothing().when(notificationHelper).addAddressTemplateVars(any(AddressGlobalUK.class), anyMap());
+        doNothing().when(notificationHelper).addHearingPostponedTemplateVars(any(CicCase.class), anyMap());
+        hearingPostponedNotification.sendToRepresentative(data, "CN1");
+
+        //Then
+        verify(notificationService).setNotificationRequest(any(NotificationRequest.class));
+
+        verify(notificationService).sendLetter();
+    }
+
+    private CaseData getMockCaseData(LocalDate stayCaseExpDate) {
+        CicCase cicCase = CicCase.builder()
+            .fullName("fullName").caseNumber("CN1")
+            .build();
+        CaseStay caseStay = CaseStay.builder()
+            .expirationDate(stayCaseExpDate)
+            .stayReason(StayReason.OTHER)
+            .additionalDetail("addlDetail")
+            .build();
+        CaseData caseData = CaseData.builder().cicCase(cicCase).caseStay(caseStay).build();
+
+        return caseData;
+    }
+
+    private DynamicList getDynamicList() {
+        final DynamicListElement listItem = DynamicListElement
+            .builder()
```

Comment:
Was this bit not removed yesterday?
Otherwise LGTM

Reply:
In the test, this change is needed as we want to set the selected value to dynamic list.